### ISSUE TITLE
Explaination about Date added and Seeders option needed

### ISSUE THREAD

**Issue by Arturoe1** [omaralvarez/deluge-autoremoveplus#64]
The options screen is not very intuitive:
* Date added which unit is using : Days or hours. 
* What does Seeders mean: Torrents with that amount of seeders or less will be deleted or what exactly. 

The option screen is a bit confusing because the units do not change (all are Min) and there is no information about those.

**Comment by mmarquezs** [comment]
> Date added which unit is using : Days or hours.

I agree that should be more clear, but just pointing out that there is a tooltip when you hover over the field and it's days.

The other stuff nothing to add from my part.

**Comment by Arturoe1** [comment]
ok thank you.

In my case in chrome using the webui i don't see any tooltips thus the comment. 
In any case in Core.py line 85 it seems to be using hourly calculation. I may be wrong thou.

**Comment by mmarquezs** [comment]
> In my case in chrome using the webui i don't see any tooltips thus the comment.

Ah good point I keep forgetting that exists... since the only use I would make of it would be on my phone and it's not usuable for that I keep forgetting it exists.

> In any case in Core.py line 85 it seems to be using hourly calculation. I may be wrong thou.

I have been using it for a long time so I can confirm it's days, but in any case I checked the code in case I have a different version or something and it's using days, is easily identified because it does divisions by 86400 which is how many seconds is a day and is used to convert the seconds to number of days.
The examples in the main page/ README.md also indicate the days thing. Again I agree that it should be clear in the settings. 
 
All said honestly this repo is very dead... like if you want changes is easier probably to do them yourself in a fork or find somebody to do them. But anyway I guess if the author comes back to do changes he will have a list of stuff I guess.
